ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -149,6 +160,43 @@ func (s *indexerStreamingService) publishTxn(ctx context.Context, req abci.Reque
 	return nil
 }
 
+// addTokenLiquidity adds the token liquidity to the event.
+// It refers to the pooled amount of each asset after a swap event has occurred.
+func (s *indexerStreamingService) addTokenLiquidity(ctx context.Context, event *abci.Event) error {
```

Comment:
Let's please cover this with a unit test 🙏

Reply:
Same here. 

I've created a [draft PR](https://github.com/osmosis-labs/osmosis/pull/8544) for indexer test cases as well as Linear issues [IND-91](https://linear.app/osmosis/issue/IND-91/add-unit-test-cases-for-adjusttokeninamountbyspreadfactor-in-ingester) and [IND-90](https://linear.app/osmosis/issue/IND-90/add-unit-test-cases-for-addtokenliquidity-in-ingester) to track them. 

Since staging and production environment are dependent on this fix and we want to receive Dexscreener feedback asap if any, if you don't mind and okay with the above arrangement, please approve this PR and I'm going to merge this PR while I'm working on the test cases. Thanks!

Reply:
Hi @p0mvn , while we will be developing test cases on different track/pace, could you please approve this PR so that we can include this for the next rebuild of indexer staging/production env? Thanks!

Reply:
Discussed offline and agreed to cherry-pick commits on v25.x branch to keep things moving with Alex and DexScreener testing while also adding tests in this PR pre-merge 🙏

Reply:
Unit test cases developed for AddTokenLiquidity & AdjustTokenInAmountBySpreadFactor